deutsche-bank-adapter/src/main/java/de/adorsys/xs2a/gateway/service/impl/DeutscheBankPaymentService.java

```diff
@@ -69,75 +36,33 @@ private void requireSepaCreditTransfer(String paymentProduct) {
         }
     }
 
-    private String writeValueAsString(SinglePaymentInitiationBody body) {
-        try {
-            return objectMapper.writeValueAsString(body);
-        } catch (JsonProcessingException e) {
-            throw new UncheckedIOException(e);
-        }
-    }
-
-    private <T> T readValue(InputStream inputStream, Class<T> klass) {
-        try {
-            return objectMapper.readValue(inputStream, klass);
-        } catch (IOException e) {
-            throw new UncheckedIOException(e);
-        }
-    }
-
     @Override
-    public SinglePaymentInitiationInformationWithStatusResponse getSinglePaymentInformation(String paymentProduct, String paymentId, Headers headers) {
+    public SinglePaymentInitiationInformationWithStatusResponse getSinglePaymentInformation(
+            String paymentProduct, String paymentId, Headers headers) {
         requireSepaCreditTransfer(paymentProduct);
 
         String uri = PAYMENTS_SEPA_CREDIT_TRANSFERS_URI + SLASH_SEPARATOR + paymentId;
 
         Map<String, String> headersMap = headers.toMap();
-        addRequiredExtraHeadersForGet(headersMap);
-
-        return httpClient.get(uri, headersMap, okResponseHandler(SinglePaymentInitiationInformationWithStatusResponse.class));
-    }
-
-    private void addRequiredExtraHeadersForGet(Map<String, String> headersMap) {
-        headersMap.put(DATE_HEADER_NAME, DateTimeFormatter.RFC_1123_DATE_TIME.format(ZonedDateTime.now()));
-        headersMap.put("Accept", "application/json");
```

Comment:
This header is necessary for get requests but it's absent on the right side.

Reply:
it was moved to abstract class

Reply:
Only extra headers for post requests were moved.

Reply:
At the moment they both are identical. I propose to split between different request types when you really will faced on. If we see that they are different we'll create new one. We should agree inside the team.
But for now i've already created separate method

Reply:
They are not identical. You're setting `Content-Type` instead of `Accept` for get requests.

Reply:
Sorry my bad, fixed